doc/examples/transform/plot_fundamental_matrix.py

```diff
@@ -0,0 +1,57 @@
+"""
+=============================
+Fundamental matrix estimation
+=============================
+
+This example demonstrates how to robustly estimate epipolar geometry between two
+views using sparse ORB feature correspondences.
+
+The fundamental matrix relates corresponding points between a pair of
+uncalibrated images. The matrix transforms homogeneous image points in one image
+to epipolar lines in the other image.
+
+Uncalibrated means that the intrinsic calibration (focal lengths, pixel skew,
+principal point) of the two cameras is not known. The fundamental matrix thus
+enables projective 3D reconstruction of the captured scene. If the calibration
+is known, estimating the essential matrix enables metric 3D reconstruction of
+the captured scene.
+
+"""
+from skimage import data
+from skimage.color import rgb2gray
+from skimage.feature import match_descriptors, ORB, plot_matches
+from skimage.measure import ransac
+from skimage.transform import FundamentalMatrixTransform
+import matplotlib.pyplot as plt
+
+img1, img2 = map(rgb2gray, data.middlebury2014_motorcycle())
```

Comment:
I'd rename this data element, it's too wordy imho. How about `stereo_motorcycle`? The citation can live in the docstring.